core/genesis.go

```diff
@@ -429,7 +429,7 @@ func (g *Genesis) ToBlock() *types.Block {
 
 	extraData := g.ExtraData
 	if g.Config.AstriaOverrideGenesisExtraData {
-		extraData = g.Config.AstriaExtraData()
+		extraData = g.Config.AstriaExtraData(1)
```

Comment:
what is the `1` here for?

Reply:
height 1 because there is no named fork at height 0, they start at 1.